lib/src/day_based_changable_picker.dart

```diff
@@ -179,6 +179,7 @@ class _DayBasedChangeablePickerState<T>
                         padding: widget.datePickerLayoutSettings.contentPadding,
                         child: _buildMonthNavigationRow()),
                   ),
+                  SizedBox( height: 40, width: double.infinity,),
```

Comment:
Why height is 40?
I assume it should not be hardcoded.
Also if we made new version with such changes all previous users will get this padding. Which is not good. 
Seems like solution should not involve breaking changes.